### ISSUE TITLE
why the downchannel should send directives interface every seconds 

### ISSUE THREAD

**Issue by kangshifu** [raj10071997/Alexa-Voice-Service#9]
in your project,we send directives interface to amazon serve evert seconds,when we catch IOException,we send the serve request immediately in Downchannel.java

**Comment by raj10071997** [comment]
Yes, I thought if the call to enqueue fails then rather than starting again I should make the call again to that method and send the request again to amazon server. Is this what you are asking?
